docker/common/install_base.sh

```diff
@@ -34,11 +34,15 @@ apt-get install -y --no-install-recommends \
 apt-get clean
 rm -rf /var/lib/apt/lists/*
 # setup gcc
-add-apt-repository ppa:ubuntu-toolchain-r/test
-apt-get update
-apt-get install -y --no-install-recommends libcilkrts5 gcc-$GCC_VERSION g++-$GCC_VERSION
-update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-$GCC_VERSION 50
-update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-$GCC_VERSION 50
+if [[ "$GCC_VERSION" == 4.9 ]]; then
```

Comment:
SO this installs from PPA when asked for GCC 4.9, but what is the version shipped with the system?

Reply:
system version 4.8.4

Reply:
then I don't really understand how is this supposed to make it use gcc-5.4 as the commit message says

Reply:
for the trusty distrib, we use gcc4.9 but default shipped is 4.8.4
for xenial, we use gcc 5.4 but default shipped by ubuntu toolchain is 5.5 as of April 24

Reply:
so how do we get gcc 5.4 then?  we only install from ppa if 4.9 is requested.  so when 5.4 is requested -- nothing happens.  I assume than the default 5.5 is used, which sort of contradicts your commit message.

Reply:
if gcc version 4.9 is requested, we get it from ppa. otherwise, don't use the toolchain and apt install it. Earlier, gcc5 was coming from ppa as well which used to be gcc5.4 before April 24

Reply:
I guess the part that is missing from the description is that xenial has 5.4 _by default_

Reply:
got it. I understand now what you mean. I'll edit the description. :)

Reply:
thanks! sorry it took so long